torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -228,6 +228,49 @@ def from_float(
             dtype=input_float.dtype
         )
 
+    @classmethod
+    def from_float_static(
+        cls,
+        input_float: torch.Tensor,
+        scale: torch.Tensor,
+        zero_point: torch.Tensor,
+        block_size: Tuple[int, ...],
```

Comment:
what's the full type definition for this?

Reply:
you mean `block_size`? this is just a tuple of variable length of ints